extensions/wikia/AdEngine/js/GptSraHelper.js

```diff
@@ -0,0 +1,45 @@
+/*global define,setTimeout*/
+/*jshint maxlen:125, camelcase:false, maxdepth:7*/
+define('ext.wikia.adEngine.gptSraHelper', [
+	'ext.wikia.adEngine.gptHelper',
+	'wikia.log'
+], function (gptHelper, log) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.gptSraHelper',
+		flushed = false,
+		sraSlots = [
+			'CORP_TOP_LEADERBOARD',
+			'HOME_TOP_LEADERBOARD',
+			'HUB_TOP_LEADERBOARD',
+			'INVISIBLE_SKIN',
+			'TOP_LEADERBOARD'
+		];
+
+	function shouldFlush(slotName) {
```

Comment:
At the beginning I liked those small functions which gave us logging and had descriptive names. But since you removed one of them I can see the benefits of removing the other one as well :smiling_imp: 

I think I can see now what @gabrys saw :japanese_ogre: If we remove this method we could get rid of `flushed` variable as well and use the `sraSlots.indexOf(slotName) === -1` condition directly in `pushAd()` function :smile_cat: We'll gain:
* one variable less,
* less code,
* and we won't loose logging since we can always add logging in `pushAd()` :)